README.md

```diff
@@ -202,6 +202,9 @@ You can specify the browser and Node.js versions by queries (case insensitive):
 * `cover 99.5%`: most popular browsers that provide coverage.
 * `cover 99.5% in US`: same as above, with [two-letter country code].
 * `cover 99.5% in my stats`: uses [custom usage data].
+* `supports rtcpeerconnection`: browsers with support for specific features.
+  `rtcpeerconnection` here is the `feat` parameter at the URL of the [Can I Use]page. A list of all available features can be found at
```

Comment:
```suggestion
  `rtcpeerconnection` here is the `feat` parameter at the URL of the [Can I Use] page. A list of all available features can be found at
```

Reply:
Good catch, I've missed it just in the new line wrapping :+1:, fixed.